modules/tutorials/pages/java-tutorial/adding-course-enrollments.adoc

```diff
@@ -0,0 +1,44 @@
+= Add Course Enrollments
+:description: Add enrollment information to the student records using the Couchbase SDK.
+:page-topic-type: tutorial
+:page-pagination: full
+:page-toclevels: 2
+:imagesdir: ../../images
+
+[abstract]
+{description}
+
+== Add Enrollment Details
+
+To add enrollment details to a student record:
+
+. In your `java` directory, create a new file called `AddEnrollments.java`.
+. Paste the following code block into your `AddEnrollments.java` file:
++
+[source, java]
+----
+include::example$AddEnrollments.java[]
+----
++
+NOTE: Because this is a tutorial, you do not need to add an error check to make sure that your collection has returned an item.
+In a live application, error checks must be made to prevent errors and keep the application running.
++
+. Open a terminal window and navigate to your `student` directory.
```

Comment:
Tell users explicitly that they need to replace the username, password, and connection string.

```suggestion
. In the `AddEnrollments.java` file, replace the `<<connection-string>>`, `<<username>>`, and `<<password>>` placeholders with the connection string, username, and password that you noted when configuring the cluster connection.
. Open a terminal window and navigate to your `student` directory.
```